mutate/mutate_fuzzer.go

```diff
@@ -0,0 +1,179 @@
+// +build gofuzz
+/*
```

Comment:
You need to add one more blank space between `// +build` and the other comment. Build comments are super particular about whitespace.